martini-build-image/codebuild.tf

```diff
@@ -0,0 +1,44 @@
+resource "aws_codebuild_project" "codebuild_name" {
+  name        = "${var.environment}-${var.pipeline_name}-codebuild"
+  description = "CodeBuild project for Martini"
+
+  environment {
+    compute_type                = "BUILD_GENERAL1_MEDIUM"
+    image                       = "aws/codebuild/amazonlinux2-x86_64-standard:5.0"
+    type                        = "LINUX_CONTAINER"
+    image_pull_credentials_type = "CODEBUILD"
+    privileged_mode             = false
+
+    environment_variable {
+      name  = "PARAMETER_NAME"
+      value = aws_ssm_parameter.martini_build_images.name
+      type  = "PLAINTEXT"
```

Comment:
Are you sure about that? :)

Reply:
Hi Rafal,

I’ve addressed all the points you raised in my latest code change except for this one. I'd like to clarify, the variable values are set as SecureString. Only the parameter name is in `PLAIN_TEXT` because it’s coded directly in the buildspec file. However, this doesn’t expose the parameter’s variable or values, as they remain securely stored as `SecureString`.

Reply:
Are you sure? https://docs.aws.amazon.com/codebuild/latest/APIReference/API_EnvironmentVariable.html

Reply:
That makes sense `The name or key of the environment variable.`. Thank you for pointing that out. I'll address this in my next code change.

Reply:
Hi @rafaljanicki. I've applied the requested changes.

Reply:
@jasonquinto I see you've removed that env variable. Is that ok? Is it not needed anymore?

Reply:
Yep, I really wracked my brain over this, but I finally realized two things:

1. It’s bad practice, as it will appear in the environment variables tab when you run a build in CodeBuild.
2. It’s redundant, as the buildspec already handles this.